src/modules/direction/store/directionSlice.ts

```diff
@@ -34,7 +34,7 @@ export interface IDirectionState {
   };
   materials: IMaterialsState;
   filters?: {
-    [key in DirectionFilterTypes]?: IFilter;
+    [key in FilterTypeEnum]?: IFilter;
   };
```

Comment:
should be a part of material state since it's related to the materials not to the direction itself